src/controllers/auth.controller.ts

```diff
@@ -556,7 +556,10 @@ export const googleOauthHandler = async (req: Request, res: Response) => {
 			return res.status(StatusCodes.NOT_FOUND).redirect(FRONTEND_URL);
 		}
 
-		// TODO: if account is already added than redirect to frontend
+		// if account is already added than redirect to frontend
+		if (foundUser.connected_services.filter((s) => s.service === "google").length) {
```

Comment:
don't check with service check the email because we can connect multiple google accounts